content/docs/integrations/snyk/index.md

```diff
@@ -49,13 +44,28 @@ Click Save.
 
 Wait a few moments for the secret to be applied.
 
-### Step 2: Add the relevant Snyk annotations to a component
+### Step 2: Add the Organization name to Roadie
+
+Components need `snyk.io/org-id` annotation set in order to work properly. Setting correct organisation name will automatically retrieve organization id and use it as a value for this annotation. 
```

Comment:
Small copy change: "Setting the correct organisation" rather than "Setting correct organisation"